examples/official-storybook/stories/addon-docs/forward-ref.stories.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { DocgenButton } from '../../components/DocgenButton';
 
-const ForwardedButton = React.forwardRef((props, ref) => <DocgenButton ref={ref} {...props} />);
+const ForwardedButton = React.forwardRef((props = { label: '' }, ref) => (
```

Comment:
docs attempts to render this component without any props, so this adds some default props. If none given it would give an propType error.